src/encoded/docs/schema-changes.md

```diff
@@ -154,87 +154,208 @@ This new object is an array of example objects that can successfully POST agains
 Updating an existing schema
 ----------------
 
-There are 2 categories when we talk about updating an existing schema:
+There are two situations we need to consider when updating an existing schema: (1) No update of the schema version (2) Update schema version
 
-* No affect on schema version
-* Update schema version
+**When not to update a version**
 
+* Do not update the schema version if the updated schema allows all existing objects in the database to continue to validate. For example, a new enum value in an existing list of enums would not cause existing objects of that type to fail validation.
```

Comment:
Since this sounds very black and white when it actually isn’t, it’d be helpful to mention the exception to the rule noted below. Someone might read this and conclude they don’t need to update the version number without reading on to find the exception.